test/extended/cli/version.go

```diff
@@ -0,0 +1,101 @@
+package cli
+
+import (
+        "encoding/json"
+        "fmt"
+        "log"
+        "os"
+        "regexp"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+)
+
+var _ = g.Describe("[cli]oc version[Conformance]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc                     = exutil.NewCLI("oc", exutil.KubeConfigPath())
+	)
+
+	g.Describe("oc version before login", func() {
+		g.It("check version without login", func() {
```

Comment:
Please follow the [naming rules](https://docs.google.com/document/d/1k761p65J0Ig81hwZWaw73QJV6SBa7kPxbjuTEBlBbF4/edit#heading=h.nvqhnby7zbfg), thanks!